contributed_definitions/nyaml/NXfit_region.yaml

```diff
@@ -0,0 +1,92 @@
+category: base
+doc: |
+  Description of region for peak fitting (i.e., in an NXpeak_model).
+  
+  Used for descibring a subpart of the abscissa (in 2D spectra) that is
+  used for fitting.
+type: group
+NXfit_region(NXobject):
+  label(NX_CHAR):
+    doc: |
+      Human-readable label which specifies which concept/entity
+      the region represents/identifies.
+  start(NX_NUMBER):
+    unit: NX_ANY
+    doc: |
+      Minimal value of the defined region.
+  stop(NX_NUMBER):
+    unit: NX_ANY
+    doc: |
+      Minimal value of the defined region.
+  (NXbackground):
+    doc: |
+      Fitted background (functional, position, and intensities).
+  (NXcollection):
+    doc: |
+      In the case of an analytical description (or if peak_model is other) this
+      collection holds parameter of (and eventually) the functional form.
+      For example in the case of Gaussians mu, sigma, cut-off values,
+      and background intensity are relevant parameter.
```

Comment:
Why should a region have an analytical form? I don't understand this here.

Reply:
Right, this was copied unintentionally.